src/tinypeel/tinypeel.py

```diff
@@ -28,6 +28,14 @@ def runPeelingCycles(pedigree, peelingInfo, args, singleLocusMode=False):
                     pedigree.AAP[mfx] = np.full(
                         peelingInfo.nLoci, 0.5, dtype=np.float32
                     )
+                else:
+                    AAP = pedigree.AAP[mfx]
+                    for i in range(peelingInfo.nLoci):
```

Comment:
@RosCraddock @XingerTang in R, for loops are slow and vectorised work with vectors is much faster. Is it the same in Python? I am just mindful if we have many loci if such bits will add to compute time a lot - not this bit on it’s own, but collectively. We probably want to optimise for speed later if and when that becomes an issue, so just thinking out loud again;)

Reply:
I think it is the same in Python!